packages/frontend/src/components/shared/NoteFloatModal.vue

```
@@ -161,4 +161,4 @@ onBeforeUnmount(() => {
       ></div>
     </div>
   </div>
-</template>
+</template>
```

ESLint

[error] Insert `⏎`

(prettier/prettier)